ci/clean_trailing_spaces.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 # trailing whitespaces
-sed -i -E 's#\s+$##g' config/set/*/*.yaml docs/*.md README.md
+sed -i '' -E 's#\\s+$##g' config/set/*/*.yaml docs/*.md README.md
```

Comment:
This now keeps failing on Linux.

Travis didn't fail, because the check was not added there. I've added it:
https://github.com/rectorphp/rector/pull/2322

Reply:
Fix for unix and macOs both is #2323